src/toJSONSchema/schemas.ts

```diff
@@ -156,4 +158,16 @@ export const SCHEMA_CONVERTERS: {
                 return { type: 'string', format: 'date-time' };
         }
     },
+    undefined(_, __, context) {
+        if (!context.undefinedStrategy) {
+            throw new Error('The "undefinedStrategy" option must be set to handle date validators');
+        }
+
+        switch (context.undefinedStrategy) {
+            case 'any':
+                return SCHEMA_CONVERTERS['any'];
+            case 'custom':
+                return {}
```

Comment:
Looks like `custom` is exactly like `any`. Maybe we should only have the `any` strategy for now ?